Project.toml

```diff
@@ -14,14 +14,17 @@ REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
 
 [compat]
+Compat = "2.1"
```

Comment:
I don't think you can put compat on extras right now.

Reply:
I just did :laughing: But yeah, best to remove if it does not have any effect on the resolver.

It's again just for `isnothing` though, so might be worth dropping it altogether here as well?

Reply:
Yea, I like `=== nothing` better anyway.